testkit/src/api.rs

```diff
@@ -41,6 +43,37 @@ pub enum ApiKind {
     Service(&'static str),
 }
 
+#[derive(Debug)]
+struct TestkitApiError(ApiError);
+
+impl Into<TestkitApiError> for Response {
```

Comment:
You should not implement `Into` where `From` is available. I can't see the reason why you can't use `From` here, but maybe I'm mistaken?

Reply:
Oh, okay, I see the reason now, so nevermind